examples/react/eslint-legacy/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@tanstack/query-example-react-basic",
```

Comment:
I think this is the problem: multiple examples with the same name!

Reply:
fixed that here: c836a9533464791f9f6f6204a9ff891d6314eb9f

Reply:
Oh thanks 🤦